com.unity.render-pipelines.universal/Shaders/BakedLit.shader

```diff
@@ -249,7 +251,7 @@ Shader "Universal Render Pipeline/Baked Lit"
             #pragma shader_feature_local _NORMALMAP
             #pragma shader_feature_local_fragment _SURFACE_TYPE_TRANSPARENT
             #pragma shader_feature_local_fragment _ALPHATEST_ON
-            #pragma shader_feature_local_fragment _ALPHAPREMULTIPLY_ON
+            #pragma shader_feature_local_fragment _ _ALPHAMODULATE_ON
```

Comment:
Not an issue, but the _ is not necessary for shader feature when there are only two keywords. Just mentioning because it's inconsistent with the other pass in same shader (line 54)